recipes-kernel/linux/linux-mainline/openvario-43-rgb.dts

```diff
@@ -26,3 +26,10 @@
 	pins = "PI10", "PI11";
 	function = "uart5";
 };
+
+// UART6 on Schematic - enabling this removes SPI functionality in order to enable 4th IGC port usage
+&uart6 {
+    pinctrl-names = "default";
+    pinctrl-0 = <&uart6_pi_pins>;
+    status = "okay";
+};
```

Comment:
Do you really need to copy-paste this in all `dts` file? Can we put it to `openvario-common.dts` instead?

Reply:
OK, 

- Moved the code to openvario-common.dts
- The description/instructions are in the commit message